dio_cache_interceptor/lib/src/dio_cache_interceptor_cache_utils.dart

```diff
@@ -124,6 +124,13 @@ extension _DioCacheInterceptorUtils on DioCacheInterceptor {
     return options.keyBuilder(
       url: request.uri,
       headers: request.getFlattenHeaders(),
+      data: switch (request.data) {
```

Comment:
I was too hurry last time, `request.data` should is fine since it will be an Object after all.

Reply:
Ok ok so we don't need the switch ?